workflows/postprocessing.nf

```diff
@@ -120,7 +120,7 @@ process genotypeGVCF {
     script:
     def familyId = meta.familyId
     def args = task.ext.args ?: ''
-    def argsjava = task.ext.args ?: ''
+    def argsjava = task.ext.argsjava ?: ''
```

Comment:
Bug fix. For now, I fixed only this process, but the bug is probably present in many others. I will create a github issue so that we don't forget this.